web/rootfs/defaults/settings-config.js

```diff
@@ -313,8 +313,6 @@ config.deploymentInfo.userRegion = '{{ $DEPLOYMENTINFO_USERREGION }}';
 
 
 // Testing
-//
```

Comment:
LEave these please, I've used that style of comments to designate a "section"

Reply:
ok